flashchips.c

```diff
@@ -16441,6 +16441,38 @@ const struct flashchip flashchips[] = {
 		.voltage	= {1650, 1950}, /* Fast read (0x0B) and multi I/O supported */
 	},
 
+	{
+		.vendor		= "Winbond",
+		.name		= "W25X05",
+		.bustype	= BUS_SPI,
+		.manufacture_id	= WINBOND_NEX_ID,
+		.model_id	= WINBOND_NEX_W25X05,
+		.total_size	= 64,
+		.page_size	= 256,
+		.feature_bits	= FEATURE_WRSR_WREN,
+		.tested		= TEST_OK_PREW,
+		.probe		= probe_spi_rdid,
+		.probe_timing	= TIMING_ZERO,
+		.block_erasers	=
+		{
+			{
+				.eraseblocks = { {4 * 1024, 16} },
+				.block_erase = spi_block_erase_20,
+			}, {
+				.eraseblocks = { {32 * 1024, 2} },
+				.block_erase = spi_block_erase_52,
+			}, {
+				.eraseblocks = { {64 * 1024, 1} },
+				.block_erase = spi_block_erase_d8,
+			}
+		},
+		.printlock	= spi_prettyprint_status_register_plain,
+		.unlock		= spi_disable_blockprotect,
+		.write		= spi_chip_write_256,
+		.read		= spi_chip_read,
+		.voltage	= {2500, 3300},
```

Comment:
Are you sure that these voltages are correct?

The datasheet mentions the following:
![kép](https://user-images.githubusercontent.com/1609182/64860925-e6c4af00-d62e-11e9-8109-e7edec538018.png)

![kép](https://user-images.githubusercontent.com/1609182/64860856-baa92e00-d62e-11e9-9f42-7dc5e0deef45.png)

At the moment if I know correctly these values are unused in flashrom, but I once the NI-845x patch get in it will be used for limiting the voltage of specific programmers.

Reply:
Good catch. I think that they are incorrect. Of the two choices, which pair ((min=2.3, max=2.7) or (min=2.7, max=3.6)) would be preferred for the values in .voltage?

Reply:
The datasheet is indeed a crap one, but here: 
![kép](https://user-images.githubusercontent.com/1609182/68495753-25eb3700-0251-11ea-87e7-617042ac2afe.png)
has the opposite direction of this table.

Basically what I understand from this is the fact that the maximum clock speed is dependent on the supply voltage. (Higher speeds require higher supply voltage.)

I would suggest to set the power range to 2.3 to 3.6 V range, since this is the actual true range. As far as I know fast read instruction is not yet supported by flashrom, and most programmers won't work faster than 50 MHz.

Reply:
OK. I've updated the voltage ranges to the values that you suggested.